bnd.bnd

```diff
@@ -2,4 +2,7 @@
 Provide-Capability:  osgi.service;objectClass:List<String>="org.apache.sling.installer.api.OsgiInstaller,org.apache.sling.installer.api.ResourceChangeListener,org.apache.sling.installer.api.info.InfoProvider,org.apache.sling.installer.api.tasks.RetryHandler",\
                      osgi.service;objectClass:List<String>="org.apache.sling.installer.api.event.InstallationListener,org.apache.sling.installer.api.jmx.InstallerMBean"
 -conditionalpackage: org.apache.felix.cm.json.*
-Bundle-DocURL: https://sling.apache.org/documentation/bundles/osgi-installer.html
\ No newline at end of file
+Bundle-DocURL: https://sling.apache.org/documentation/bundles/osgi-installer.html
+# to make compatible with older Commons Johnzon (https://issues.apache.org/jira/browse/SLING-9218) remove the require-capability header
+-plugin: org.apache.sling.bnd.plugin.headers.parameters.remove.Plugin;\
+    'Require-Capability'='osgi.contract;osgi.contract=JavaJSONP;filter:="(&(osgi.contract=JavaJSONP)(version=1.1.0))"'
```

Comment:
@oliverlietz Do you know why this is not working? I took https://github.com/apache/sling-org-apache-sling-resourceresolver/commit/8a610d0b4cd55fa6949b92faf28eb0a64ad938a4 as blueprint....